pkg/controller/common/api/v1/common.go

```diff
@@ -0,0 +1,158 @@
+package v1
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// RelativeObjectMap holds object related to given parent object.
+// The structure is [GroupVersionKind] -> [common.relativeName] -> *unstructured.Unstructured
+// where:
+//   GroupVersionKind - identifies type stored in entry
+//   relativeName() - return path to object in relation to parent
```

Comment:
nit: can we provide a sample/example here

Reply:
query: Has namespace anything to do w.r.t this map?

Reply:
yes, it depends also on parent type